<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamewindow</class>
 <widget class="QMainWindow" name="gamewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-image:url(:/image/background.png);
}
QPushButton {
    background-color: #FFD700;
    color: black;
    border: 2px solid #B8860B;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    font-family: 'Arial';
    margin: 4px 2px;
    transition-duration: 0.4s;
    cursor: pointer;
}
QPushButton:hover {
    background-color: #B8860B;
    color: white;
}
QLabel {
    color: white;
    font-size: 14px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="_2">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="horizontalSpacing">
       <number>8</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="PlayerLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>182</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>182</height>
         </size>
        </property>
        <property name="text">
         <string>Player:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="PScoreLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>33</width>
          <height>182</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>33</width>
          <height>182</height>
         </size>
        </property>
        <property name="text">
         <string>Score</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="DScoreLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>33</width>
          <height>182</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>33</width>
          <height>182</height>
         </size>
        </property>
        <property name="text">
         <string>Score</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Blackjack Symulator Â© 2024 / 2025                                                                                                                                                                                   github.com/Mateusz-Latka</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Casino&quot;;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:700; color:#ffff00;&quot;&gt;BLACKJACK SYMULATOR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="HitButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>700</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Hit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="DealerLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>182</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>182</height>
         </size>
        </property>
        <property name="text">
         <string>Dealer:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="StandButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>700</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Stand</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <layout class="QHBoxLayout" name="DealerLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
        </property>
       </layout>
      </item>
      <item row="2" column="2" colspan="2">
       <layout class="QHBoxLayout" name="PlayerLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
